src/js/notion.js

```diff
@@ -49,9 +49,60 @@ export default class Notion {
     }
   }
 
+  async checkDuplicateEntry(paperId, databaseId) {
+    const entries = await this.retrieveEntry(paperId, databaseId);
+    if (entries.length != 0) {
+      document.dispatchEvent(
+        new CustomEvent('msg', {
+          detail: {
+            type: 'warning',
+            msg: 'This item is already bookmarked. Opening existing entry...',
+          },
+        })
+      );
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+    }
+    return entries;
+  }
+
+  async retrieveEntry(paperId, databaseId) {
+    const filter = {
+      property: 'URL',
+      rich_text: {
+        contains: `/${paperId}`,
```

Comment:
wonder if an ID conflicts with the other one🤔 
https://info.arxiv.org/help/arxiv_identifier_for_services.html